### PR TITLE
Revert "Pimcore\Tool\Console: Remove deprecated methods"

### PR DESCRIPTION
**Pull request by mcop1**
Reverts pimcore/pimcore#14024
We need to find a replacement for `runPhpScriptInBackground` first (and maybe others too).

### CODE REVIEW

**Comment by blankse** [comment]
@mcop1 Is the replacement not the symfony messenger queue?

**Comment by mcop1** [comment]
> @mcop1 Is the replacement not the symfony messenger queue?

@blankse Yes, it should be. But unfortunately it is not that easy in some bundles. We will try to find a solution until the release of Pimcore 11, but for now we have to revert the changes- sorry.

File: lib/Tool/Console.php [hunk 1]
```diff
@@ -115,6 +139,26 @@ public static function getExecutable(string $name, bool $throwException = false)
...
+            if (strpos($process->getOutput() . $process->getErrorOutput(), 'imagemagick.org') !== false) {
+                return true;
+            }
+        } catch (\Exception $e) {
+            // noting to do
+        }
+
+        return false;
+    }
+
```
Comment (blankse) [inline lib/Tool/Console.php]: This is not used, right?
```suggestion
```

Comment (mcop1) [reply lib/Tool/Console.php]: Right, as far as I can see they are not used.

**Pull request merged**